packages/atlas/src/components/CategoryLink/CategoryLink.styles.ts

```diff
@@ -0,0 +1,41 @@
+import styled from '@emotion/styled'
+import { Link } from 'react-router-dom'
+
+import { SkeletonLoader } from '@/components/_loaders/SkeletonLoader'
+import { sizes } from '@/styles'
+import { square } from '@/styles'
+import { hexToRgb } from '@/styles/utils'
+
+type ContainerProps = {
+  disabled?: boolean
+}
+
+export const Container = styled(Link)<ContainerProps>`
+  display: inline-flex;
+  align-items: center;
+  text-decoration: none;
+  pointer-events: ${({ disabled }) => (disabled ? 'none' : 'auto')};
+`
+
+type IconWrapperProps = {
+  color: string
+  withHandle: boolean
+}
+
+export const IconWrapper = styled.div<IconWrapperProps>`
+  ${square(40)}
+  border-radius: 50%;
+  background-color: ${({ color }) => `rgba(${hexToRgb(color)}, 0.2) `};
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-right: ${({ withHandle }) => (withHandle ? sizes(3) : 0)};
+
+  path {
+    fill: ${({ color }) => color};
+  }
+`
```

Comment:
We have the same element in VideoCategoryCard. Can we just export this and use it here? 
![image](https://user-images.githubusercontent.com/51168865/152745099-dc212d77-a226-4b6d-a261-a284d61e6e54.png)